lib/widgets/message_list.dart

```diff
@@ -480,12 +486,13 @@ class MarkAsReadWidget extends StatelessWidget {
                 backgroundColor: _UnreadMarker.color,
                 minimumSize: const Size.fromHeight(38),
                 textStyle:
-                  // Restate [FilledButton]'s default, which inherits from
-                  // [zulipTypography]…
-                  Theme.of(context).textTheme.labelLarge!
+                  // Restate [FilledButton]'s default…
+                  themeData.textTheme.labelLarge!.merge(filledButtonThemeTextStyle)
```

Comment:
Huh.

Does this mean we could just set `.textTheme.labelLarge` on our main ThemeData, and skip reaching into all the individual button styles? That seems like it'd be cleaner.

In principle that has a slightly broader effect than specified in #548, in that it affects any other Material widgets that use the `labelLarge` text style. But fundamentally those are widgets that the Material designers intended to have a similar text style to that of buttons (even though they're not called "buttons"), so it's likely that that'll be the right answer anyway.

If we end up with such a widget where Vlad wants a different spacing, we can address that then… and I think it's fairly likely that in that case he'd want something that wasn't the Material widget in any event.

Reply:
That's a good idea! Done.